### PR TITLE
More HMIS app settings, support resolving logo

### PR DESCRIPTION
**Pull request by gigxz**

### CODE REVIEW

File: drivers/hmis/app/controllers/hmis/app_settings_controller.rb [hunk 1]
```diff
@@ -10,8 +10,24 @@ class Hmis::AppSettingsController < Hmis::BaseController
...
+    hostname = ENV['FQDN']
+
     render json: {
       oktaPath: okta_enabled ? '/hmis/users/auth/okta' : nil,
+      logoPath: logo_path.present? ? ActionController::Base.helpers.asset_path(logo_path) : nil,
+      warehouseUrl: "https://#{hostname}",
+      warehouseName: _('Boston DND Warehouse'),
+      resetPasswordUrl: "https://#{hostname}/users/password/new",
+      unlockAccountUrl: "https://#{hostname}/users/unlock/new",
+      casUrl: nil, # TODO: get CAS url from env
```
Comment (gigxz) [inline drivers/hmis/app/controllers/hmis/app_settings_controller.rb]: Needed if we want to keep the link to CAS on the top bar (currently only enabled in QA). @eanders should I just add an env var for it?

Comment (eanders) [reply drivers/hmis/app/controllers/hmis/app_settings_controller.rb]: not sure exactly what you need, but you should be able to determine if we need to show the CAS link if `ENV['DATABASE_CAS_DB']` isn't empty.  CAS url should be available via `Config.get(:cas_url)` Would those be sufficient?

File: drivers/hmis/app/controllers/hmis/app_settings_controller.rb [hunk 2]
```diff
@@ -10,8 +10,24 @@ class Hmis::AppSettingsController < Hmis::BaseController
 
   def show
     okta_enabled = ENV['HMIS_OKTA_CLIENT_ID'].present? && ENV['OKTA_DOMAIN'].present?
+
+    logo = ENV['LOGO']
+    logo_path = if SerializedAsset.exists?(logo)
+      SerializedAsset.get_src(logo)
+    else
+      "theme/logo/#{logo}"
```
Comment (eanders) [inline drivers/hmis/app/controllers/hmis/app_settings_controller.rb:18]: HMIS may want a different logo than the warehouse, but this seems fine for now (until I manage to get the admin for theming in place.)

**Review by eanders — APPROVED** [review]

File: drivers/hmis/app/controllers/hmis/app_settings_controller.rb (hunk 1 above)
Comment (gigxz) [reply drivers/hmis/app/controllers/hmis/app_settings_controller.rb]: Yes, thanks

**Pull request merged**